### PR TITLE
feat: add `getIdentitiesByPublicKeyHashes` method

### PR DESCRIPTION
**Pull request by jawid-h**
## Issue being fixed or feature implemented

Adding  `getIdentitiesByPublicKeyHashes` method

## What was done?

Added  `getIdentitiesByPublicKeyHashes` method

## Breaking Changes

None.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: lib/methods/platform/getIdentitiesByPublicKeyHashesFactory.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+/**
+ * @param {GrpcTransport} grpcTransport
+ * @returns {getIdentitiesByPublicKeyHashes}
+ */
+function getIdentitiesByPublicKeyHashesFactory(grpcTransport) {
+  /**
+   * Fetch the identities by public key hashes
+   *
+   * @typedef {getIdentitiesByPublicKeyHashes}
+   * @param {string[]} publicKeyHashes
```
Comment (shumkov) [inline lib/methods/platform/getIdentitiesByPublicKeyHashesFactory.js:19]: Let's accept buffers

File: lib/methods/platform/getIdentityIdsByPublicKeyHashesFactory.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+/**
+ * @param {GrpcTransport} grpcTransport
+ * @returns {getIdentityIdsByPublicKeyHashes}
+ */
+function getIdentityIdsByPublicKeyHashesFactory(grpcTransport) {
+  /**
+   * Fetch the identities by public key hashes
+   *
+   * @typedef {getIdentityIdsByPublicKeyHashes}
+   * @param {string[]} publicKeyHashes
```
Comment (shumkov) [inline lib/methods/platform/getIdentityIdsByPublicKeyHashesFactory.js:19]: Buffer[]

**Review by shumkov — APPROVED** [review]
👍

**Review by shumkov — APPROVED** [review]
👍

**Comment by Alex-Werner** [comment]
Don't forget: https://dashevo.github.io/js-dapi-client/#/

**Pull request merged**